.github/workflows/reusable-prober.yml

```diff
@@ -61,27 +60,14 @@ jobs:
       sigstore_probe: ${{ steps.msg.outputs.sigstore_probe }}
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Set up Go
-        id: setup-go
-        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v3.3.0
-        with:
-          go-version-file: 'prober/hack/toolz/go.mod'
-          check-latest: true
-          cache: false
+      - uses: imjasonh/setup-crane@00c9e93efa4e1138c9a7a5c594acd6c75a2fbf0c # v0.3
 
-      - uses: actions/cache@704facf57e6136b1bc63b828d79edcd491f0ee84 # v3.3.2
-        with:
-          path: |
-            ~/.cache/go-build
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ steps.setup-go.outputs.go-version }}-${{ github.job }}-${{ hashFiles('prober/hack/toolz/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-${{ steps.setup-go.outputs.go-version }}-${{ github.job }}-
-
-      - name: Install 'prober' from sigstore/scaffolding
+      - name: Copy probe from image
         run: |
-          make -C prober/ prober
-          echo "PATH=$PATH:$PWD/prober/hack/toolz/bin" >> $GITHUB_ENV
+          crane export $PROBE_IMAGE - | tar -xf - -C /tmp /ko-app/prober
+          cp /tmp/ko-app/prober /usr/local/bin
+        env:
+          PROBE_IMAGE: ghcr.io/sigstore/scaffolding/prober:v0.6.9@sha256:1b745626f5603698442ed55f4bdeb1a86f2192b603ccc3343750a3b70304ff94
```

Comment:
should this also be listed in a fake Dockerfile (and extracted here) so that Dependabot will see it and flag updates for it?